app/views/clientsmodal.html

```diff
@@ -57,6 +57,14 @@ <h4 class="modal-title" id="addClientLabel">
         </div>
       </div>
 
+      <div class="form-group" ng-class="{ 'has-error' : ngError.certFingerprint }">
+        <label><span>**</span> Client certificate</label>
```

Comment:
I'd rather leave it as it I think, I don't think it should be capitalised, its just a noun.